pom.xml

```diff
@@ -322,6 +322,25 @@
         </pluginManagement>
 
         <plugins>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.5</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
+                        <phase>test</phase>
```

Comment:
I'm not sure it makes sense to put this in the test phase.. What about the verify phase instead?

Reply:
Or what about no phase at all and this is only run adhoc in travis?

Reply:
Just so anybody can get some context, [from the maven docs](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html):

> test - test the compiled source code using a suitable unit testing framework. These tests should not require the code be packaged or deployed
> verify - run any checks on results of integration tests to ensure quality criteria are met

In Jacoco, the test phase seems to be used in the default templates, and this seems a good value to me. Verify could be used in case we want to meet a certain criteria, maybe if we want to ensure we have a minimum code coverage of XX%?